DemoServer/Controllers/Demos/JavascriptIndexes/JavascriptMapIndex/metadata.json

```diff
@@ -0,0 +1,161 @@
+{
```

Comment:
There is no notion of language (c#) this example is implemented in, at the top of left column.
Is it because there is just one language covered so far, so single lang is not displayed?

Reply:
yes - IIRC - when there is only the C# demo then the app doesn't show the languages